infrastructure/iac/template/index.html

```diff
@@ -96,4 +96,8 @@ <h2>References</h2>
 </div>
 </body>
 ${ideScript}
```

Comment:
Here we can have a self invoked function to encapsulate all logic and have only one script tag like in IntelliJ, this would require minimal changes to how other IDE's inject the script.